Makefile.in

```diff
@@ -121,5 +119,23 @@ install:
 	$(INSTALL) -m 644 $(SRC_LDFILE) $(DESTDIR)$(target_libdir)/$(LDFILE)
 
 sinclude .deps
-.deps:
+.deps: bfd-headers/.stamp
 	$(CC) -MM $(CPPFLAGS) $(srcdir)/*.c > .deps
+
+
+# Augment the path (relative or absolute) to create a symlink from 1 path
+# component lower. Make has no easy way to check if a value does *not*
+# start with a slash, hence this trick: first, prepend "../" unconditionally
+# and if a path was absolute, it will now start with "..//" which will be
+# replaced back with a single "/".
+symlink_path1 = $(patsubst ..//%,/%,$(patsubst %,../%,$(1)))
```

Comment:
why do you need to do this ?  in case someone passes a bfd path like ../ ?

you can do this instead in the shell code:

`bfddir=$(cd "$(BFD_INCLUDE_DIR)"; pwd)`

then you'll have access to `$$bfddir` everywhere.

alternatively, update the configure.ac code to do that:

`bfd_include_dir="$(cd "$ac_bfd_include_dir"; pwd)"`